pydra/src/pydra/tasks/mrtrix3/_version.py

```diff
@@ -0,0 +1,4 @@
+# file generated by setuptools_scm
+# don't change, don't track in version control
+__version__ = version = '3.0.5.dev589+g9e2806609.d20230817'
+__version_tuple__ = version_tuple = (3, 0, 5, 'dev589', 'g9e2806609.d20230817')
```

Comment:
Are these files intended to be included in the PR? Certainly the header comment suggests not.